reflekt-plugin/src/test/kotlin/io/reflekt/plugin/ir/IrTransformFunctionsTest.kt

```diff
@@ -1,112 +1,107 @@
-                ),
-                Signature("(FunctionTestClass) -> Unit",  "IrTestAnnotation::class"),
-                "FunctionTestClass()"
-            ),
-            Arguments.of(
-                setOf(
-                    "fun io.reflekt.test.ir.FunctionTestClass.fun2(): kotlin.Int"
-                ),
-                Signature("(FunctionTestClass) -> Int", "IrTestAnnotation::class"),
-                "FunctionTestClass()"
-            ),
-            Arguments.of(
-                setOf(
-                    "fun io.reflekt.test.ir.FunctionTestClass.fun3(): kotlin.collections.List<kotlin.Int>"
-                ),
-                Signature("(FunctionTestClass) -> List<Int>", "IrTestAnnotation::class"),
-                "FunctionTestClass()"
-            ),
-            Arguments.of(
-                setOf(
-                    "fun io.reflekt.test.ir.FunctionTestClass.fun4(kotlin.Int, kotlin.Float?, kotlin.collections.Set<kotlin.Boolean>): kotlin.collections.List<kotlin.String>"
-                ),
-                Signature("(FunctionTestClass, Int, Float?, Set<Boolean>) -> List<String>", "IrTestAnnotation::class"),
-                "FunctionTestClass(), 1, null, emptySet()"
-            ),
-        )
-
-        @JvmStatic
-        fun getSmartReflektFunctionsTestData(): List<Arguments> = listOf(
-            Arguments.of(setOf(""),
-                Signature("() -> Boolean", "it.name == \"fooBoolean\""),
-                ""
-            )
-        )
-
-    }
-
-    @ParameterizedTest(name = "Function test#{index} with [{arguments}]")
-    @MethodSource("getReflektFunctionsTestData")
-    fun testReflektFunctions(expectedFunctions: Set<String>, functionsSignature: Signature, functionsArguments: String) {
-        Assertions.assertEquals(expectedFunctions, ReflektType.REFLEKT.functionsInvokeCall(functionsSignature, functionsArguments).call(false))
-    }
-
-    @ParameterizedTest(name = "Member function test#{index} with [{arguments}]")
-    @MethodSource("getReflektMemberFunctionsTestData")
-    fun testReflektMemberFunctions(expectedFunctions: Set<String>, functionsSignature: Signature, functionsArguments: String, ) {
-        Assertions.assertEquals(expectedFunctions, ReflektType.REFLEKT.functionsInvokeCall(functionsSignature, functionsArguments).call(false))
-    }
-
-    @ParameterizedTest(name = "Function test#{index} with [{arguments}]")
-    @Disabled("Failed with IllegalStateException, but in examples everything works fine")
-    @MethodSource("getSmartReflektFunctionsTestData")
-    fun testSmartReflektFunctions(expectedFunctions: Set<String>, functionsSignature: Signature, functionsArguments: String) {
-        Assertions.assertEquals(expectedFunctions, ReflektType.SMART_REFLEKT.functionsInvokeCall(functionsSignature, functionsArguments).call())
-    }
-}
+// TODO: don't use kotlin-compile-testing since the compiler directory output is empty
```

Comment:
should we rather delete this file or exclude it from tests?

Reply:
It was fixed in the next PRs